modules/ROOT/pages/_partials/data-sync/server-using-mutations-with-a-data-sync-server-and-a-data-sync-client.adoc

```diff
@@ -0,0 +1,122 @@
+[id="server-using-mutations-with-a-data-sync-server-and-a-data-sync-client-{context}"]
+= Using Mutations with a Data Sync Server
+
+Build a Data Sync Server and query the mutation.
+The Data Sync Server will be running locally in this procedure.
+
+.Prerequisites
+
+* link:https://github.com/jstaffor/mobile-docs/blob/master/modules/ROOT/pages/_partials/sync/server-getting-started.adoc[Getting Started with Hello World Data Sync].
+* You are familiar with {sync-server} operations, for example, stopping and starting {sync-server}.
```

Comment:
{sync-server} resolves to voyager, maybe avoid using attributes?

Reply:
:+1: